scripts/zones/Caedarva_Mire/IDs.lua

```diff
@@ -45,6 +45,25 @@ zones[xi.zone.CAEDARVA_MIRE] =
     },
     mob =
     {
+        CHIGOES               =
+        {
+            ['Wild_Karakul'] =
+            {
+                17100806,
```

Comment:
Same pattern works here.. ish

```
INSERT INTO `mob_spawn_points` VALUES (17100806,'Chigoe','Chigoe',5,237.115,-8.949,-352.812,20);
INSERT INTO `mob_spawn_points` VALUES (17100807,'Chigoe','Chigoe',5,314.347,-8.709,-446.945,110);
INSERT INTO `mob_spawn_points` VALUES (17100808,'Chigoe','Chigoe',5,318.988,-10.787,-455.924,43);
INSERT INTO `mob_spawn_points` VALUES (17100809,'Chigoe','Chigoe',5,318.605,-10.5,-453.678,232);
INSERT INTO `mob_spawn_points` VALUES (17100810,'Chigoe','Chigoe',5,239.320,-8.268,-353.714,25);
INSERT INTO `mob_spawn_points` VALUES (17100811,'Chigoe','Chigoe',5,270.933,-20.099,-328.418,152);
INSERT INTO `mob_spawn_points` VALUES (17100812,'Chigoe','Chigoe',5,270.459,-20.098,-328.045,151);
INSERT INTO `mob_spawn_points` VALUES (17100813,'Chigoe','Chigoe',5,0,0,0,0);
INSERT INTO `mob_spawn_points` VALUES (17100814,'Chigoe','Chigoe',5,0,0,0,0);
INSERT INTO `mob_spawn_points` VALUES (17100815,'Chigoe','Chigoe',5,0,0,0,0);
```

I'm not sure if that 2nd group of 5 are instantiated because their x,y,z co-ords are all zero? I don't remember the rules around that. 

For the first group, you can look up with
```
['Wild_Karakul'] = GetTableOfIDs("Chigoe", 5)
```

The second group isn't so easy, you could do `{ GetFirstID('Chigoe') + 5, ... }`, but that's a bit ugly. You could go into the definition of `GetTableOfIDs` and make it take an optional offset after the optional range I suppose, to make it possible to say: `GetTableOfIDs("Chigoe", 5, 5)` : take 5 after an offset of 5

Reply:
> I'm not sure if that 2nd group of 5 are instantiated because their x,y,z co-ords are all zero? I don't remember the rules around that.

we just didn't have them capped, those are for the mosshorn at north end of mire

Reply:
oh that reminds me: make sure the chigs set to spawn by mobskill are set unspawned in db, so that they aren't "up" before the skill pops them

Reply:
I've changed them to type scripted (128)